BUILDING.md

```diff
@@ -417,41 +417,50 @@ To build the documentation:
 
 This will build Node.js first (if necessary) and then use it to build the docs:
 
-```console
-$ make doc
+```bash
+make doc
 ```
 
 If you have an existing Node.js build, you can build just the docs with:
 
-```console
-$ NODE=/path/to/node make doc-only
+```bash
+NODE=/path/to/node make doc-only
 ```
 
-To read the documentation:
+To read the man page:
 
-```console
-$ man doc/node.1
+```bash
+man doc/node.1
 ```
 
-If you prefer to read the documentation in a browser,
-run the following after `make doc` is finished:
+If you prefer to read the full documentation in a browser, run the following
+after `make doc` is finished:
 
-```console
-$ make docopen
+```bash
+make docserve
+```
+
+This will provide a URL to where you may browse the documentation locally.
+
+If you prefer to read the documentation as a one-page version of all the
```

Comment:
I'm not sure this sentence is accurate, AFAIK `make docopen` let's you navigate all the docs, not just the one-page version.

Reply:
Well, it opens the one-page version. What makes this method unique? For me it pulls up Lynx text-based browser or something.

Reply:
Thinking about this a little bit more&hellip;

It's entirely possible that my `make docopen` simply opens the docs in one's _default browser_.

On Debian WSL my default is likely one of these text-based browsers which is why it is showing up the way that it is.

Refs: https://wiki.debian.org/WebBrowsers#Default_Web_Browser

Reply:
Now beginning to wonder if there is a point in having two similar methods of serving the docs:

- `docopen`: Opens in the default browser
- `docserve`: Starts static file web server and provides URL

@aduh95, what do you think? I personally have no use for `docopen`&hellip;

Reply:
> It's entirely possible that my make docopen simply opens the docs in one's default browser.

Yes that's correct, see [`webbrowser` docs for reference](https://docs.python.org/3/library/webbrowser.html).

> what do you think? I personally have no use for docopen

I think we should keep it because it's been there forever (ade8cf96e0f19fab240b2a7fcf970e8b1d24d10b), removing it may disrupt collaborators habits without any upside.